pkg/services/featuremgmt/registry.go

```diff
@@ -262,5 +262,10 @@ var (
 			Description: "Replaces whitelabeling with the new custom branding feature",
 			State:       FeatureStateAlpha,
 		},
+		{
+			Name:        "fullstoryUserTracking",
+			Description: "Enables Fullstory cliend SDK to do user analytics",
+			State:       FeatureStateAlpha,
```

Comment:
```suggestion
```

Alpha flags requires dev_mode to be enabled which we don't on the instances you want to test this on. It's mostly a meta data about the feature toggle anyway :)

Reply:
Awesome! I didn't know that :)